### ISSUE TITLE
Sphinx doc references via intersphinx

### ISSUE THREAD

**Issue by algoterranean** [grantjenks/python-diskcache#158]
Hello. Does this project have a Sphinx `objects.inv` exposed anywhere that we can use to reference this module's docs?

**Comment by grantjenks** [comment]
Never heard of that. Pull request welcome ☺️

**Comment by grantjenks** [comment]
I make a venv, install the requirements.txt, then cd to docs and “make html”.

If all looks good, I rsync to my server.

It’s scripted in https://github.com/grantjenks/python-gj/blob/master/gj/packages.py

**Comment by algoterranean** [comment]
Ah, it looks like you're already taking care of everything then. Sphinx generates an `objects.inv` file in your `build/html`, which you upload to your server. Which is located at http://www.grantjenks.com/docs/diskcache/objects.inv. This is what I was using before when I tested it out, but I was using https, and there seems to be a domain issue with your SSL cert, which is why I ran into issues.

In any case, you're good to go. For example, I can add this to my sphinx `conf.py`:

```
interphinx_mapping = {
    'diskcache': ('http://www.grantjenks.com/docs/diskcache/', None)
}
```
and it will pick up your docs correctly.

Thanks!
